cuda_bindings/cuda/bindings/_internal/nvvm_windows.pyx

```diff
@@ -0,0 +1,373 @@
+# Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+# SPDX-License-Identifier: LicenseRef-NVIDIA-SOFTWARE-LICENSE
+#
+# This code was automatically generated across versions from 11.0.3 to 12.8.0. Do not modify it directly.
+
+from libc.stdint cimport intptr_t
+
+from .utils cimport get_nvvm_dso_version_suffix
+
+from .utils import FunctionNotFoundError, NotSupportedError
+
+import os
+import site
+
+import win32api
+
+
+###############################################################################
+# Wrapper init
+###############################################################################
+
+LOAD_LIBRARY_SEARCH_SYSTEM32     = 0x00000800
+LOAD_LIBRARY_SEARCH_DEFAULT_DIRS = 0x00001000
+LOAD_LIBRARY_SEARCH_DLL_LOAD_DIR = 0x00000100
+cdef bint __py_nvvm_init = False
+cdef void* __cuDriverGetVersion = NULL
+
+cdef void* __nvvmVersion = NULL
+cdef void* __nvvmIRVersion = NULL
+cdef void* __nvvmCreateProgram = NULL
+cdef void* __nvvmDestroyProgram = NULL
+cdef void* __nvvmAddModuleToProgram = NULL
+cdef void* __nvvmLazyAddModuleToProgram = NULL
+cdef void* __nvvmCompileProgram = NULL
+cdef void* __nvvmVerifyProgram = NULL
+cdef void* __nvvmGetCompiledResultSize = NULL
+cdef void* __nvvmGetCompiledResult = NULL
+cdef void* __nvvmGetProgramLogSize = NULL
+cdef void* __nvvmGetProgramLog = NULL
+
+
+cdef inline list get_site_packages():
+    return [site.getusersitepackages()] + site.getsitepackages()
+
+
+cdef load_library(const int driver_ver):
+    handle = 0
+
+    for suffix in get_nvvm_dso_version_suffix(driver_ver):
+        if len(suffix) == 0:
+            continue
+        dll_name = "nvvm64_40_0"
```

Comment:
TODO: check this on windows

Reply:
I downloaded

* nvidia_cuda_nvcc_cu11-11.8.89-py3-none-win_amd64.whl
* nvidia_cuda_nvcc_cu12-12.0.76-py3-none-win_amd64.whl
* nvidia_cuda_nvcc_cu12-12.8.61-py3-none-win_amd64.whl

from

* https://pypi.org/project/nvidia-cuda-nvcc-cu11/#history
* https://pypi.org/project/nvidia-cuda-nvcc-cu12/#history

Then:
```
$ for whl in *.whl; do unzip -l $whl | grep dll; done
 18079744  09-21-2022 18:38   nvidia/cuda_nvcc/nvvm/bin/nvvm64_40_0.dll
 18210304  10-25-2022 03:43   nvidia/cuda_nvcc/nvvm/bin/nvvm64_40_0.dll
 52869120  01-16-2025 05:07   nvidia/cuda_nvcc/nvvm/bin/nvvm64_40_0.dll
```

I.e. **the nvvm dll_name for 11.8 is the same as for the 12.x series.**